changelog/where.dd

```diff
@@ -0,0 +1,15 @@
+`~/dlang/install.sh` now supports `--where`
+
+`--where` has been added as a new option to the install script.
+It offers a convenient way to get the path of a specific compiler.
```

Comment:
`path of a specific compiler` is a bit vague -- `--where` doesn't provide the path to the compiler, it provides the directory where the compiler has been installed.